crates/bevy_sprite/src/mesh2d/mesh.rs

```diff
@@ -637,16 +635,12 @@ impl<P: PhaseItem, const I: usize> RenderCommand<P> for SetMesh2dBindGroup<I> {
         mesh2d_bind_group: SystemParamItem<'w, '_, Self::Param>,
         pass: &mut TrackedRenderPass<'w>,
     ) -> RenderCommandResult {
-        let mut dynamic_offsets: [u32; 1] = Default::default();
-        let mut offset_count = 0;
-        if let Some(dynamic_offset) = item.dynamic_offset() {
-            dynamic_offsets[offset_count] = dynamic_offset.get();
-            offset_count += 1;
-        }
+        let offset = [item.dynamic_offset().map_or(0, |offset| offset.get())];
+        let has_offset = item.dynamic_offset().is_some();
```

Comment:
This would end up being rewritten as soon as you need maybe more than one dynamic offset though. So I would prefer to keep consistency with SetMeshBindGroup where there are multiple as I kind of expect to combine more of the 2D and 3D things to avoid duplications. So, please remove this hunk.